tools/docker/docker-installation.md

```diff
@@ -8,12 +8,39 @@ order: 1
 
 ## Installation
 
-On Fedora 22 and above install the `docker` package:
+On Fedora 30 and above install the `docker-ce` package using the Docker repository:
 
+> for older versions of Fedora see: [docker-installation-fedora-22-29](./docker-installation-fedora-22-29.html)
+
+To install the dnf-plugins-core package (which provides the commands to manage your DNF repositories) and set up the stable repository.
+
+```
+$ sudo dnf -y install dnf-plugins-core
+```
+To add the `docker-ce` repository
+
+```
+$ sudo dnf config-manager \
+    --add-repo \
+    https://download.docker.com/linux/fedora/docker-ce.repo
```

Comment:
I take it it is not packaged in Fedora, then?

Well, could we at least mention podman somewhere? As an alternative, that is already in Fedora.

Reply:
Not that I found. I have not used podman. I will look at it.

Reply:
I added a section for alternatives on the about page. see https://github.com/developer-portal/content/pull/336/commits/78f3a8fa089a092ca1e6c1ea861b2c0b640ad3fb

Reply:
Thanks! Well podman can run rootless, it's stable, and it has some very interesting+useful features (on top of the basic ones, offered by Docker). But keeps CLI compatible with docker (at least the one I know of). I can highly recommend it.

Reply:
I looked into it. So far it seems to offer parity with what I have done in docker. Thanks for pointing it out.